src/main/fc/rc.c

```diff
@@ -278,8 +278,7 @@ static void checkForThrottleErrorResetState(uint16_t rxRefreshRate)
     static int index;
     static int16_t rcCommandThrottlePrevious[THROTTLE_BUFFER_MAX];
 
-    const int rxRefreshRateMs = rxRefreshRate / 1000;
-    const int indexMax = constrain(THROTTLE_DELTA_MS / rxRefreshRateMs, 1, THROTTLE_BUFFER_MAX);
```

Comment:
In SITL when using UDP RC input, the rxRefreshRate is smaller than 1000, thus rxRefreshRateMs be zero and then crash. I fixed it here to avoid it.